app/views/finance/bank_transactions/show.html.haml

```diff
@@ -0,0 +1,39 @@
+%p
+  %b= heading_helper(BankTransaction, :external_id) + ':'
+  = @bank_transaction.external_id
```

Comment:
I know we have this style elsewhere, I'd prefer some more declarative style with classes, or a real table. Could also be a bootstrap form with static 'input's. Unless styled, it somewhat hurts my eyes :s

Reply:
i agree, but i don't want to start a new style for now. we could do this later too